angles.py

```diff
@@ -2,18 +2,31 @@
 #import numpy as np
 import matplotlib.pyplot as mpl
 
-def sin_calc(start, end, step):
+def draw():
+    wave_calc("sin", -10, 10, 1)
+    wave_calc("cos", -10, 10, 1)
+    wave_calc("tan", -10, 10, 1)
+    mpl.show()
+
+def wave_calc(type, start, end, step):
     y_values = []
     x_values = []
-    for x in range(start * 10, end * 10, step):
-        x_val = x/10
-        x_values.append(x_val) 
-        sin_val = math.sin(x_val)
-        y_values.append(sin_val)
+    for x in range(start * 100, end * 100, step):
+        x_val = x/100
+        
+        if type == 'sin':
+            y_val = math.sin(x_val)
+        elif type =='cos':
+            y_val = math.cos(x_val)
+        else:
+            y_val = math.tan(x_val)
+        if abs(y_val) < 3:
+            x_values.append(x_val) 
+            y_values.append(y_val)
     # Draw  horizontal axis
     mpl.plot([-5.1,5.1],[0,0])
```

Comment:
You should remove these 2 lines and generate the axis using matplotlib libraries. Something like a grid would do the trick. Also, the grid should be added in the draw() function. Here it gets added every time.